packages/workflows-sdk/src/utils/composer/create-step.ts

```diff
@@ -167,8 +167,7 @@ function applyStep<
             const invokeResult =
               stepOutput?.__type ===
               OrchestrationUtils.SymbolWorkflowStepResponse
-                ? stepOutput.compensateInput &&
-                  JSON.parse(JSON.stringify(stepOutput.compensateInput))
+                ? stepOutput.compensateInput && stepOutput.compensateInput
```

Comment:
Suggestion : maybe do a deepCopy to be sure we dont have references issues unless it is expected ?

Reply:
yeah, good call. Will change both to use a deepCopy